tests/unit/tls_test.cc

```diff
@@ -410,9 +410,8 @@ SEASTAR_TEST_CASE(test_abort_accept_after_handshake) {
         b.set_x509_trust_file(certfile("catest.pem"), tls::x509_crt_format::PEM).get();
 
         auto c = tls::connect(b.build_certificate_credentials(), addr).get0();
-        server.abort_accept(); // should not affect the socket we got.
-
         auto s = sa.get0();
+        server.abort_accept(); // should not affect the socket we got.
```

Comment:
I think this points to a bug in our io_uring integration. In real life server.abort_accept() is not synchronized with calls to accept(). abort is called when shutting down, and accept is called by some tcp server trying to service new connections.

Reply:
agreed, it is not synchronized. but IMHO, neither is it guaranteed that the "accepted" event is always delivered if the listening fd accepts a connection and then gets closed right away. i *think* io_uring processes the events in batch.

anyway, will take a closer look to see how/if we can fix this in the io_uring backend.

Reply:
You're right that regardless of a bug in the io_uring implementation, this tls_test is buggy too.